changes.d/pr-384.toml

```diff
@@ -0,0 +1,53 @@
+[[scuffle-h264]]
+category = "refactor"
+description = "AVCDecoderConfigurationRecord now uses actual SpsExtended struct (we intentionally do not let it use the Sps struct)"
+breaking = true
+authors = ["@philipch07"]
+
+[[scuffle-h264]]
+category = "refactor"
+description = "Refactored Sps to be its own module with each struct having its own file"
+breaking = true
+authors = ["@philipch07"]
+
+[[scuffle-h264]]
+category = "fix"
+description = "Fixes a bug with the AVCDecoderConfigurationRecord size not being computed correctly"
+breaking = true
+authors = ["@philipch07"]
+
+[[scuffle-h264]]
+category = "fix"
+description = "Fixes a bug with the AVCDecoderConfigurationRecord not being built correctly due to byte misalignment"
+breaking = true
+authors = ["@philipch07"]
+
+[[scuffle-h264]]
+category = "feat"
+description = "All sps structs can now be \"built\", that is, they can now go from a struct back into a bytestream"
+breaking = false
+authors = ["@philipch07"]
+
+[[scuffle-h264]]
+category = "feat"
+description = "Building an Sps will automatically insert necessary emulation prevention bytes (which are already properly handled in the existing parse function)"
+breaking = false
+authors = ["@TroyKomodo"]
+
+[[scuffle-h264]]
+category = "feat"
+description = "As a result of being able to build, we also have the ability to generate minimal and functionally equivalent Sps headers via `Sps::reduce()`"
+breaking = false
+authors = ["@philipch07"]
```

Comment:
These can be removed, and replaced with something talking about the `EmulationBytesIo` thingy